index.html

```diff
@@ -586,17 +606,72 @@ <h2>
         with the <a>default display mode</a> (<code>browser</code>) being the
         last item in the chain.
       </p>
+      <p>
+        For more advanced usages, the <a data-link-for="WebAppManifest">
+        display_override</a> member can be used to specify a custom fallback
+        order. This member is valid IFF <a data-link-for="WebAppManifest">display</a> is also present.
+      </p>
+      <p>
+        The algorithm for determing the web app's requested display mode is:
+        <ol>
+          <li>[=list/For each=] <var>considering_display_mode</var> of <var>
+            <a data-link-for="WebAppManifest">display_override</a></var>:
+            <ol>
+              <li>
+                If the user agent supports the |considering_display_mode|,
+                then that display mode is used (exit).
+              </li>
+            </ol>
+          </li>
+          <li>
+            Let |considering_display_mode| be the the value specified by the
+            <a data-link-for="WebAppManifest">display</a> property.
+          </li>
+          <li>
+            If the user agent supports the |considering_display_mode|, then
+            that display mode is used (exit).
+          </li>
+          <li>
+            While |considering_display_mode| has <a>fallback display mode</a>,
+            assign that to |considering_display_mode| and go back to step 3.
+          </li>
+        </ol>
+        This algorithm will end with the final <a>fallback display mode</a> of
+        <code>browser</code>, which the user agent MUST support.
+      </p>
+      </p>
       <div class="example">
         <p>
-          For example, SuperSecure Browser (a fictitious browser) only supports
+          Example: SuperSecure Browser (a fictitious browser) only supports
           the <code>minimal-ui</code> and <code>browser</code> display modes,
           but a developer declares that she wants <code>fullscreen</code> in
-          the manifest. In this case, the user agent will first check if it
+          the manifest by using the <a data-link-for="WebAppManifest">display
```

Comment:
```suggestion
          the manifest by using the {{WebAppManifest/display}}
```

Reply:
Nit: Same change globally. Avoid using `data-link-for=` as we now support the new syntax. Quick overview of ReSpec shorthands (and docs): https://respec.org/docs/#Shorthands-Guide

Reply:
done - let me know if there are other places you see that you would like me to update :)
or if you see links that I did incorrectly